playbooks/roles/custom_certificates/tasks/main.yml

```diff
@@ -87,3 +87,6 @@
       -out "{{ custom_certificates_ca_directory_certs }}/{{ custom_certificates_server }}.crt"
   args:
     creates: "{{ custom_certificates_ca_directory_certs }}/{{ custom_certificates_server }}.crt"
+
+- include: issue.yml
+  when: custom_certificates_hostname is defined
```

Comment:
Not my intention to hold this PR up, but if the above does work I'd prefer that to running this multiple times per hostname.